theming.js

```diff
@@ -139,25 +150,26 @@ var ThemeManager = new Lang.Class({
     },
 
     _updateDashColor: function() {
-        if (this._settings.get_boolean('custom-background-color')) {
-            let [backgroundColor, borderColor] = this._getDefaultColors();
+        let [backgroundColor, borderColor] = this._getDefaultColors();
```

Comment:
A comment here would be very helpful, as the way we tweak the colour is a bit messy :)
1. We get the default dash background (inherithed from the theme)
2. If customized we use the cutsom background color, but maintain the opacityof the default one
3. If the opacity is customized, we also get the custom opacity from the settings
4. The opacity is eventually overwritten by the transparency machinery

Reply:
I added some comments here [5e1790435553318c29d6b705a25bb16699d72694]. Not sure if you find this is enough though.